ScanbotSDKExamples/NET/ReadyToUseUI.Droid/Utils/TextDataScanner.cs

```diff
@@ -1,6 +1,10 @@
-﻿namespace ReadyToUseUI.Droid.Utils
+﻿
+
+using static IO.Scanbot.Sdk.UI.View.Generictext.Entity.TextDataScannerStep;
+
+namespace ReadyToUseUI.Droid.Utils
 {
-    public class ValidationCallback : Java.Lang.Object //, IGenericTextRecognizerGenericTextValidationCallback
+    public class ValidationCallback : Java.Lang.Object, IGenericTextValidationCallback
```

Comment:
@yurii-scanbot, since ValidationCallback and RecognitionCallback are both only one method classes, I would suggest adding extension methods and wrapper classes in the binding library to make it possible to pass a C# delegate to the classes that need these callbacks as it simplifies usage for the consumer.